components/registration/TabProj/TabProj.module.css

```diff
@@ -0,0 +1,25 @@
+.root {
```

Comment:
if you have styles that are exactly the same, you can have a common stylesheet in the shared directory
in fact, that would be preferred because one change would change both

Reply:
you can also have two stylesheets simply by importing them with different names (i.e. styles vs commonStyles)

Reply:
I have separate css files for TabProj and TabShortResponse because it's likely (since the figma isn't finalized) that they will have slightly different styles going forward. Should I put a common stylesheet in the shared directory or leave as is?

Reply:
ah if you expect them to be different we can leave it as is